### PR TITLE
Add .editorconfig file

### PR DESCRIPTION
**Pull request by SplittyDev**
This pull request adds a `.editorconfig` file to help contributors adhere to the coding guidelines.
It also changes the shebang line of the `src/Build/Setup.sh` script from `#!/bin/bash` to the more portable `#!/usr/bin/env bash`. That should allow it to work on macOS and other unix-like systems to run flawlessly.

Signed-off-by: Splitty <splittydev@gmail.com>

### CODE REVIEW

File: src/Build/Setup.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (amaneureka) [inline src/Build/Setup.sh]: Could you apply same changes [here](https://github.com/amaneureka/AtomOS/tree/master/libs) too

Comment (SplittyDev) [inline src/Build/Setup.sh]: Will do

Comment (SplittyDev) [inline src/Build/Setup.sh]: Done.

**Comment by amaneureka** [comment]
Looks good to me!

**Pull request merged**